src/main/java/org/skriptlang/skript/bukkit/loottables/elements/expressions/ExprLootContextKiller.java

```diff
@@ -0,0 +1,78 @@
+package org.skriptlang.skript.bukkit.loottables.elements.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.bukkit.loot.LootContext;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.loottables.LootContextCreateEvent;
+import org.skriptlang.skript.bukkit.loottables.LootContextWrapper;
+
+@Name("Killer of Loot Context")
+@Description("Returns the loot context killer of a loot context " +
+	"Note that setting the killer of a loot context will read the player's tool enchantments (e.g. looting) when generating loot.")
+@Examples({
+	"set {_killer} to loot context killer of {_context}",
+	"",
+	"set {_context} to a loot context at player:",
+		"\tset loot context luck value to 10",
+		"\tset loot context killer to player",
+		"\tset loot context entity to last spawned pig"
+})
+@Since("INSERT VERSION")
+public class ExprLootContextKiller extends SimplePropertyExpression<LootContext, Player> {
+
+	static {
+		registerDefault(ExprLootContextKiller.class, Player.class, "loot [context] killer", "lootcontexts");
```

Comment:
```suggestion
		registerDefault(ExprLootContextKiller.class, Player.class, "(looter|looting player)", "lootcontexts");
```
same reasoning as looted entity